### PR TITLE
feat(searxng): add accent support and fix auto themes

### PR DESCRIPTION
**Pull request by ryanccn**

### CODE REVIEW

File: styles/searxng/catppuccin.user.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
 @preprocessor less
 @var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha"]
 @var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accent "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Grey"]
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:14]: This should be accentColor
```diff
- @var select accent "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Grey"]
+ @var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Grey"]
```

File: styles/searxng/catppuccin.user.css [hunk 2]
```diff
@@ -126,8 +127,8 @@ url-prefix("https://trydex.tk/searxng/"),
 url-prefix("https://www.gruble.de/"),
 url-prefix("https://www.webrats.xyz/"),
 url-prefix("https://xcxc.ml/"),
-url-prefix("https://xo.wtf/") {
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:129]: Can you remove any formatting changes like right here

File: styles/searxng/catppuccin.user.css [hunk 3]
```diff
@@ -240,10 +241,19 @@ url-prefix("https://xo.wtf/") {
       @base: #1e1e2e;
       @mantle: #181825;
       @crust: #11111b;
+    };
+  };
+
+  :root.theme-auto {
+    #catppuccin(@lightFlavor);
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:248]: you need to pass `accentColor` into here, so it should be
```diff
- #catppuccin(@lightFlavor);
+ #catppuccin(@lightFlavor, @accentColor);
```

File: styles/searxng/catppuccin.user.css [hunk 4]
```diff
@@ -240,10 +241,19 @@ url-prefix("https://xo.wtf/") {
...
+  }
+
+  @media (prefers-color-scheme: dark) {
+    :root.theme-auto {
+      #catppuccin(@darkFlavor);
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:253]: same as above

File: styles/searxng/catppuccin.user.css [hunk 5]
```diff
@@ -240,10 +241,19 @@ url-prefix("https://xo.wtf/") {
...
     }
   }
 
-  :root.theme-auto,
   :root.theme-dark {
     #catppuccin(@darkFlavor);
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:258]: same as above

File: styles/searxng/catppuccin.user.css [hunk 6]
```diff
@@ -373,16 +384,24 @@ url-prefix("https://xo.wtf/") {
...
+
     if (additions) {
       article.result {
         background-color: var(--color-result-background);
-        border-radius: .75em;
-        padding: .75em;
-        margin: .5em;
+        border-radius: 0.75em;
+        padding: 0.75em;
+        margin: 0.5em;
```
Comment (isabelroses) [inline styles/searxng/catppuccin.user.css:400]: Why does this need changing?

**Review by isabelroses — CHANGES_REQUESTED** [review]

File: styles/searxng/catppuccin.user.css (hunk 6 above)
Comment (ryanccn) [reply styles/searxng/catppuccin.user.css:400]: formatted

File: styles/searxng/catppuccin.user.css (hunk 3 above)
Comment (ryanccn) [reply styles/searxng/catppuccin.user.css:248]: Why is this necessary?

Comment (isabelroses) [reply styles/searxng/catppuccin.user.css:248]: We are just trying to enforce common practices with the userstyles and i missed the other part which is why the theme still works.

**Review by isabelroses — COMMENTED** [review]
Can you also bump the version number whilst you make these changes,

**Review by isabelroses — APPROVED** [review]

**Pull request merged**